### PR TITLE
Update about page background

### PR DESCRIPTION
**Pull request by mukhtyar**
This PR contains changes to the About page for the expanding Cal-Adapt enterprise. Changes include:
- Rename “Background” to “Our Roadmap”
- Added new paragraph on Cal-Adapt enterprise
- Add Cal-Adapt roadmap graphic

### CODE REVIEW

**Comment by mukhtyar** [comment]
[Update About page](https://trello.com/c/1QAXp66v/25-update-about-page)

**Review by melijimenez11 — APPROVED** [review]

File: src/routes/about/index.svelte [hunk 1]
```diff
@@ -549,6 +549,31 @@
...
           iterations of updates.
         </p>
+        <p>
+          Cal-Adapt is evolving substantially, expanding beyond its current
+          capabilities to support <a
+            href="https://opr.ca.gov/climate/icarp/climate-assessment/"
+            target="_blank">California’s Fifth Climate Change Assessment</a
+          >
+          and offer a more comprehensive and powerful solution for highly technical
+          & data-intensive needs, with a focus on the energy sector. The expanded
```
Comment (elehmer) [inline src/routes/about/index.svelte]: Again maybe replace the '&' with 'and'

**Review by elehmer — APPROVED** [review]
Do we want to render the roadmap PNG just a little larger? My old eyes are having trouble with the small green text.

File: src/routes/about/index.svelte (hunk 1 above)
Comment (mukhtyar) [reply src/routes/about/index.svelte]: Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/256/commits/32c481bd1634ca33062de3ee9a0ab98528807f34

**Comment by mukhtyar** [comment]
> Do we want to render the roadmap PNG just a little larger? My old eyes are having trouble with the small green text.

I increased the font size a little and darkened the italicized text. Let me know if that's better or same. I'm taking a screenshot of the Google Slide, not sure how to get a higher res image than the current size. Exporting from Google Slides doesn't help.

**Pull request merged**